### PR TITLE
Update multiple cluster doc

### PR DESCRIPTION
**Pull request by klaasdebeuf**

### CODE REVIEW

**Comment by MariekeGO** [comment]
This addition isn't entirely clear to me. What do you mean with "across all the configurations"?

**Comment by MichielMN** [comment]
@MariekeGO it means that the ID needs to be the same on any agent for a specific DB, <DBServer>https://xyz.skyline.local</> needs to have ID = 0 on all agents, not ID =0 on one and ID = 1 on another because this will mess up password encryption

**Comment by klaasdebeuf** [comment]
@MichielMN  , I added also this comment for the ID.
Could it be that the documentation for "Prior to DataMiner 10.4.0/10.4.2" is completly wrong as it doesn't mention anything about the ID but the ID was added in 10.3.10?

**Comment by MariekeGO** [comment]
> @MichielMN , I added also this comment for the ID. Could it be that the documentation for "Prior to DataMiner 10.4.0/10.4.2" is completly wrong as it doesn't mention anything about the ID but the ID was added in 10.3.10 and did cause some problems at Fox?

According to RN [37446](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm.aspx?ID=37446), the ID attribute was introduced in 10.4.0/10.4.2.

**Comment by klaasdebeuf** [comment]
> > > @MichielMN , I added also this comment for the ID. Could it be that the documentation for "Prior to DataMiner 10.4.0/10.4.2" is completly wrong as it doesn't mention anything about the ID but the ID was added in 10.3.10 and did cause some problems at Fox?
> > 
> > 
> > According to RN [37446](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm.aspx?ID=37446), the ID attribute was introduced in 10.4.0/10.4.2.
> 
> It was added in DB.xml to merge the two files in one, Klaas is correct that the attribute was introduced prior. However the ID attribute was meant to be automatically configured by Cube without manual intervention, the cube change was reverted to avoid a pre-existing issue however. So in the backend the ID attribute existed, without it being written. There was a problem where the ID attribute not existing caused dataminer to not start up but this has been fixed (can't find the exact change now). I would just make it clear that in any version that ID needs to be the same across the cluster for a specific Db

@MichielMN , I will leave it out for now as a fix was made for that.

File: user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ To configure this setup, proceed as follows:
 
    - **Password**: The password that will be used to connect to OpenSearch. This is the equivalent of the *PWD* tag in the *DB.xml* file (see [Specifying custom credentials for OpenSearch or Elasticsearch](xref:DB_xml#specifying-custom-credentials-for-opensearch-or-elasticsearch)).
 
-   - **Prefix**: The prefix for the OpenSearch indexes. This is the equivalent of the *DB* tag in the *DB.xml* file (see [Specifying a custom prefix for the indexes](xref:DB_xml#specifying-a-custom-prefix-for-the-indexes)).
+   - **Prefix**: The prefix for the OpenSearch indexes. This is the equivalent of the *DB* tag in the *DB.xml* file. The specified value should match across all the OpenSearch configurations in the *DBConfiguration.xml*. (see [Specifying a custom prefix for the indexes](xref:DB_xml#specifying-a-custom-prefix-for-the-indexes)).
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md:88]:
```suggestion
   - **Prefix**: The prefix for the OpenSearch indexes. This is the equivalent of the *DB* tag in the *DB.xml* file. The specified value must be the same for each OpenSearch cluster defined in *DBConfiguration.xml*. (see [Specifying a custom prefix for the indexes](xref:DB_xml#specifying-a-custom-prefix-for-the-indexes)).
```

File: user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/DB_xml.md [hunk 2]
```diff
@@ -553,7 +553,7 @@ To do so:
 
 1. Open the file *DB.xml* (in the folder *C:\\Skyline DataMiner\\*).
 
-1. In the *DB* element for the indexing database, specify the custom prefix. Keep in mind that only regular alphanumeric characters are supported for the prefix, not symbols.
+1. In the *DB* element for the indexing database, specify the custom prefix. Keep in mind that only regular alphanumeric characters are supported for the prefix, not symbols. The specified value should match across all the OpenSearch configurations in the *DB.xml*.
```
Comment (MariekeGO) [inline user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/DB_xml.md:556]:
```suggestion
1. In the *DB* element for the indexing database, specify the custom prefix. Keep in mind that only regular alphanumeric characters are supported for the prefix, not symbols. The specified value must be the same for each OpenSearch cluster defined in *DB.xml*.
```

File: user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ To configure this setup, proceed as follows:
 
    - *PWD*: The password to connect to OpenSearch.
 
-   - *DB*: The prefix for the OpenSearch indexes.
+   - *DB*: The prefix for the OpenSearch indexes. The specified value should match across all the OpenSearch configurations in the *DBConfiguration.xml*
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md:35]: DBConfiguration.xml is no longer used from DataMiner 10.4.0/10.4.2 onwards, so I'd like to change this as follows:

```suggestion
   - *DB*: The prefix for the OpenSearch indexes. The specified value must be the same for each OpenSearch cluster defined in *DB.xml*.
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**